MetamodelAlgemeen.md

```diff
@@ -1193,9 +1202,27 @@ degelijk heeft, maar de waarde ervan is onbekend. De kardinaliteit wordt dan
 niet van 1 naar 0 gezet, maar er wordt aangegeven dat er sprake is van mogelijk
 geen waarde. Meer hierover is beschreven in het hoofdstuk [Afspraken & Regels](#afspraken-regels).
 
+Een generalisatie is een bijzondere vorm van een relatie. De kardinaliteit van de bron en van het doel is hier altijd en per definitie 1..1. Dit hoeft daarom nooit via een aanduiding van een kardinaliteit te worden aangegeven.
+
 *Toepassing*: alle modelelementen die een kenmerk (kunnen) zijn van een
 objecttype.
 
+#### Metagegeven: **Kardinaliteit relatie bron**
+
+>   **Definitie Kardinaliteit relatie bron**  
+>   Aanduiding of het kenmerk een authentiek gegeven betreft.
+
+Voorbeeld: een verblijfsobject ligt in een pand. De eigenaar van de relatie is het verblijfsobject (de bron van de relatie) en het doel van de relatie is een pand.
+
+- De kardinaliteit van het doel van de relatie geeft aan: in hoeveel panden kan 1 verblijfsobject liggen. Antwoord: 1..*
+- De kardinaliteit van de bron geeft aan: hoeveel verblijfsobjecten kunnen er in 1 pand liggen. Antwoord: 0..*
+
+*Toelichting*
+
+De kardinaliteit van de bron van de relatie is een regel die geldt voor de data (een constraint). Wanneer de kardinaliteit van de bron van de relatie 1 of meer is, dan geeft dit aan dat het object dat het doel van de relatie is alleen kan bestaan wanneer ernaar gewezen wordt vanuit de bron van de relatie. Anders gezegd, het object dat het doel van de relatie is mag geen wees zijn en mag dit ook niet worden.
+
```

Comment:
- Wanneer de kardinaliteit aan de bron kant van de relatie niet is gespecificeerd dan is er geen sprake van een default waarde.

Reply:
ah dit staat er nog. Ik heb moeite met de tekst bij de toelichting.
Stel dit voor:
De kardinaliteit van de bron van de relatie geeft aan hoeveel instanties van de bron van de relatie kunnen verwijzen naar één instantie van het doel van de relatie. Wanneer de kardinaliteit aan de bron kant van de relatie niet is gespecificeerd dan is er geen sprake van een defaultwaarde.

Reply:
De definitie stond er nog niet:
>   **Definitie Kardinaliteit relatie bron**  
>   De kardinaliteit relatie bron geeft bij een relatiesoort aan aan hoeveel instanties van het bron objecttype in een relatie met één instantie van het doel objecttype verbonden kunnen zijn.

Reply:
ok

Reply:
verwerkt